policies/cla.yml

```diff
@@ -15,108 +15,109 @@ configuration:
       bypassOrgs:
          - bloomberg
       bypassUsers:
-         - dependabot[bot]
```

Comment:
@MIchaelMainer  is this an addition to the list, or you deleted some of the bots as well?

Reply:
@DragosDanielBoia  I just reordered the list to make reviewing it easier. There should still be the same number on the bypass list.